src/NuGet.CommandLine.StagingExtensions/StageCommand.cs

```diff
@@ -146,6 +179,144 @@ private bool TryParseArguments(out SubCommand subCommand, out string parameter)
             }
         }
 
+        private async Task GetStageDetails(StageManagementResource stageManagementResource, string stageId)
+        {
+            var stageDetails = await stageManagementResource.Get(stageId, Console);
+
+            Console.WriteLine();
+            Console.PrintJustified(0, string.Format(CultureInfo.CurrentCulture, "{0}: {1} ({2})", StagingResources.HeaderStageName, stageDetails.DisplayName, stageDetails.Id));

```

Comment:
Can we make a wrapper function for all of these `PrintJustified` calls? Something so that we could just do `PrintJustified(StagingResources.HeaderStageName, stageDetails.DisplayName, stageDetails.Id)` and so on. We could also use the function to clean up the code above as well.